test/futures/rest/account_test.exs

```diff
@@ -3,20 +3,50 @@ defmodule ExHuobi.Futures.Rest.Account.Test do
 
   use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
-  @config %ExHuobi.Config{
-    api_key: System.get_env("HUOBI_API_KEY"),
-    api_secret: System.get_env("HUOBI_API_SECRET")
-  }
+  # @config %ExHuobi.Config{
+  #   api_key: System.get_env("HUOBI_API_KEY"),
+  #   api_secret: System.get_env("HUOBI_API_SECRET")
+  # }
```

Comment:
If this code unused, we can remove it

Reply:
@kimquy I have fixed all the comment. Please help to review again. 
Thanks